sentry/const.py

```diff
@@ -44,46 +32,9 @@ def split_multiple(string, delimiter=',', strip_chars=None):
 ]
 DEFAULT_IGNORED_EXCEPTIONS = ','.join(ODOO_USER_EXCEPTIONS)
 
-PROCESSORS = (
-    'raven.processors.SanitizePasswordsProcessor',
-    'odoo.addons.sentry.logutils.SanitizeOdooCookiesProcessor',
-)
-DEFAULT_PROCESSORS = ','.join(PROCESSORS)
-
 EXCLUDE_LOGGERS = (
     'werkzeug',
 )
 DEFAULT_EXCLUDE_LOGGERS = ','.join(EXCLUDE_LOGGERS)
 
-DEFAULT_TRANSPORT = 'threaded'
-
-
-def select_transport(name=DEFAULT_TRANSPORT):
-    return {
-        'requests_synchronous': raven.transport.RequestsHTTPTransport,
-        'requests_threaded': raven.transport.ThreadedRequestsHTTPTransport,
-        'synchronous': raven.transport.HTTPTransport,
-        'threaded': raven.transport.ThreadedHTTPTransport,
-    }.get(name, DEFAULT_TRANSPORT)
-
-
-def get_sentry_options():
-    return [
-        SentryOption('dsn', '', str.strip),
-        SentryOption('install_sys_hook', False, None),
-        SentryOption('transport', DEFAULT_TRANSPORT, select_transport),
-        SentryOption('include_paths', '', split_multiple),
-        SentryOption('exclude_paths', '', split_multiple),
-        SentryOption('machine', defaults.NAME, None),
-        SentryOption('auto_log_stacks', defaults.AUTO_LOG_STACKS, None),
-        SentryOption('capture_locals', defaults.CAPTURE_LOCALS, None),
-        SentryOption('string_max_length', defaults.MAX_LENGTH_STRING, None),
-        SentryOption('list_max_length', defaults.MAX_LENGTH_LIST, None),
-        SentryOption('site', None, None),
-        SentryOption('include_versions', True, None),
-        SentryOption(
-            'ignore_exceptions', DEFAULT_IGNORED_EXCEPTIONS, split_multiple),
-        SentryOption('processors', DEFAULT_PROCESSORS, split_multiple),
-        SentryOption('environment', None, None),
-        SentryOption('release', None, None),
-    ]
+DEFAULT_ENVIRONMENT = 'production'
```

Comment:
No.
A wrongly configured instance should appear as "production".

Reply:
I've changed 'production' by 'develop'.